fdbserver/DataDistribution.actor.cpp

```diff
@@ -2844,22 +2861,25 @@ ACTOR Future<Void> waitHealthyZoneChange( DDTeamCollection* self ) {
 			state Future<Void> healthyZoneTimeout = Never();
 			if(val.present()) {
 				auto p = decodeHealthyZoneValue(val.get());
-				if(p.second > tr.getReadVersion().get()) {
+				if (p.first == ignoreSSFailure) {
+					// healthyZone is now overloaded for DD diabling purpose, which does not timeout
+					healthyZoneTimeout = Never();
```

Comment:
Should this case have a trace event like the others?

Reply:
Done. Will push a new commit once correctness finishes 100K.